types/office-js-preview/index.d.ts

```diff
@@ -580,12 +580,24 @@ declare namespace Office {
         ): Promise<RemoveEventListener>;
     }
     /**
-     * An interface that contains all the functionality provided to manage the state of the OFfice ribbon.
+     * An interface that contains all the functionality provided to manage the state of the Office ribbon.
      */
     interface Ribbon {
+        /**
+         * Registers a custom contextual tab with Office and defines the tab's controls.
+         * This method only requests that the tab be registered. The actual registration is controlled by the Office application and may not be complete when the returned Promise object is resolved.
```

Comment:
Nit: `Promise` should have code ticks around it.
```suggestion
         * This method only requests that the tab be registered. The actual registration is controlled by the Office application and may not be complete when the returned `Promise` object is resolved.
```

Reply:
done

Reply:
Fixed